src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -372,12 +372,28 @@ public Builder volumes(final String... volumes) {
     @JsonProperty("Retries")
     public abstract Integer retries();
 
-    @JsonCreator
+    /**
+     * In nanoseconds.
+     */
+    @Nullable
+    @JsonProperty("StartPeriod")
+    public abstract Long startPeriod();
```

Comment:
Can you add a javadoc saying this is only available for docker api versions 1.29+?